### ISSUE TITLE
Skin of the Earth Features

### ISSUE THREAD

**Issue by JeffWootton** [iho-ohi/S-101-Documentation-and-FC#38]
During discussions within various groups (the S-57 to S-101 Conversion Sub-Group in particular), and concerns raised by representatives of the IEHG, there is a requirement to re-open the discussion on the features in S-101 that constitute the Skin of the Earth.  The main concern is over the features in S-101 that have been added as Skin of the Earth features (DockArea and LockBasin), and the question has been asked:
- For larger scale ENC data where it is required to include depth information (DepthArea, DredgedArea, UnsurveyedArea) within a dock or lock basin, how is the information contained within the DockArea or LockBasin feature (categoryofDock, condition, horizontalClearanceFixed, horizontalClearanceLength, horizontalClearanceWidth, maximumPermittedDraught and status for DockArea; horizontalClearanceFixed, horizontalClearanceLength, horizontalClearanceWidth and status for LockBasin) going to be encoded in the data?

The guidance for DockArea (DCEG clause 8.18) and LockBasin (DCEG clause 8.20) states that where the dock/lock is navigable at the maximum display scale of the ENC, it must be encoded using the features DepthArea, DredgedArea [or Unsurveyed Area].  Given that Skin of the Earth features must not overlap; this guidance is also included in the DCEG in that the encoding of DockArea and LockBasin where the dock or lock is considered to be navigable at the maximum display scale of the ENC is prohibited.  This means that the additional information that may be required for the dock/lock cannot be encoded.

In S-57, the object classes FLODOC, HULKES and PONTON were included as Group 1 (Skin of the Earth) features.  These have been removed as Skin of the Earth features for S-101, as they do not in reality form a part of the surface of the earth.  The inclusion of DockArea and LockBasin as Skin of the Earth features in S-101 was the result of discussions within the DCEG Sub-Group in the early days of the development of the document.  The principle argument put forward at that time for their inclusion as Skin of the Earth was that at smaller scales where the dock/lock is not navigable there is no requirement for encoding underlying depth information, however at this time the implications of their inclusion in regard to encoding additional required information for the dock/lock for larger scale ENCs where the dock/lock is navigable were not realized.

In order to rectify this issue, it is proposed that the DockArea and LockBasin features are removed as Skin of the Earth features in S-101, such that the Skin of the Earth features only consist of DepthArea, DredgedArea, LandArea and UnsurveyedArea.  Resultant actions would be:
- Amendments to the encoding guidance at DCEG clauses 8.18.1 (DockArea) and 8.20.1 (LockBasin).  It is suggested that some new guidance is included for the encoding of DepthArea coincident with the dock/lock at smaller scales where the dock/lock is not navigable.  In addition, entries for DockArea and LockBasin will be required to be included in Table 2.8 (sample scaleMinimum policy).
- Amendment to S-101 Main document:  Remove DockArea and LockBasin from the list of Skin of the Earth features at clause 4.3.2.1.1.

There is no anticipated change requires for the Feature or Portrayal Catalogues.

**Comment by KlasOstergren-SMA** [comment]
I agree in the described issue and support the removal of DockArea and LockBasin as Skin of the earth features.
How detailed guidance for the DepthArea in smaller scales is really needed? For other areas not navigable in the scale, the data producer generalize the depth information and set the depth interval of the DepthArea based of the depth contours. Is it enough to state that the DockArea and LockBasin must be covered by a DepthArea?

**Comment by MikkoHovi-FI** [comment]
That one country would be Finland :) 

When this was discussed back in time, our stance was and still is, that as all navigable waters should be encoded as depth areas, by definition all RIVERS, LAKAREs etc are by definition non-navigable. And as they they are non-navigable, they are of no significant importance to the mariner and therefore should be subject to the use of SCAMIN.  And if SCAMIN is allowed then they can't be Group 1. 

In 2014 it was agreed as a compromise that RIVERS, RAPIDS and LAKAREs would stay in Group 2 and LOKBSNs and DOCAREs would be moved to Group1 as proposed by France.
